chain/events/events_called.go

```diff
@@ -464,14 +467,20 @@ type messageEvents struct {
 
 	lk       sync.RWMutex
 	matchers map[triggerID]MsgMatchFunc
+
+	blsMsgLk    sync.Mutex
+	blsMsgCache *lru.ARCCache
```

Comment:
Since BlockMessages include bls and secp messages, I'd call this something different than `bls..` (e.g `blockMsg..`)

Reply:
resolved